rviz_default_plugins/CMakeLists.txt

```diff
@@ -108,14 +115,46 @@ set(rviz_default_plugins_source_files
   src/rviz_default_plugins/view_controllers/orbit/orbit_view_controller.cpp
 )
 
-add_library(rviz_default_plugins SHARED
+add_library(rviz_default_plugins_object OBJECT
   ${rviz_default_plugins_headers_to_moc}
   ${rviz_default_plugins_source_files}
 )
+
+set_property(TARGET rviz_default_plugins_object PROPERTY POSITION_INDEPENDENT_CODE ON)
+
+target_include_directories(rviz_default_plugins_object PUBLIC
+  ${rviz_rendering_INCLUDE_DIRS}
+  ${rviz_common_INCLUDE_DIRS}
+  ${laser_geometry_INCLUDE_DIRS}
+  ${nav_msgs_INCLUDE_DIRS}
+  ${OGRE_INCLUDE_DIRS}
+  ${Qt5Widgets_INCLUDE_DIRS}
+  ${resource_retriever_INCLUDE_DIRS}
+  ${TinyXML_INCLUDE_DIRS}
+  ${urdf_INCLUDE_DIRS}
```

Comment:
This patch broke the build again when `urdf` is installed on the system. See #243 for a temp. workaround.